File: 1_Programing_Language/Python/Practice/Prac_0To120/Practice_5.py
# 老题：输出质数

number = int(input("please input a number:"))

i = 2
while i <= number:
    j = 2
    while j <= i**(1/2):
        if i % j == 0:
            break
        j += 1
    if j > i**(1/2):
        print(f"{i} ")
    i += 1

File: 4_Web/JavaScript/Practice/Practice_5.js
// 将任意进制转换为任意进制(2-36)
// 输入数据
var number = window.prompt('Input');
// 输入基底进制
var radix1 = parseInt(window.prompt('Input'));
// 输入目标进制
var radix2 = parseInt(window.prompt('Input'));
// 将输入数从基底进制转换为目标进制
console.log(parseInt(number,radix1).toString(radix2));